includes/admin/class-wc-payments-admin.php

```diff
@@ -438,4 +442,13 @@ private function is_tos_agreement_required() {
 	private static function is_account_overview_page_enabled() {
 		return get_option( '_wcpay_feature_account_overview' );
 	}
+
+	/**
+	 * Checks whether the grouped settings feature is enabled
+	 *
+	 * @return bool
+	 */
+	private static function is_grouped_settings_enabled() {
+		return get_option( 'wcpaydev_grouped_settings', '0' ) === '1';
```

Comment:
This is a minor nitpick but it looks like the [`_wcpay_feature_X` format](https://github.com/Automattic/woocommerce-payments/pull/1467/files#diff-3863b4f88d0515e080fadc56af3c5e38e942afb96e569cb000ae6a594e8bea4aR443) is used for feature flags, and `wcpaydev_X` for dev tools-specific settings. The latter seem to be less of feature flags and more of dev adjustments.

Maybe the option should be called `_wcpay_feature_grouped_settings` instead?

Reply:
That's a great point, I like it! Done in 759dfc0